README.md

```diff
@@ -1,68 +1,64 @@
-9wm
-====
+# 9wm
 
-9wm is an X11 window manager inspired by the Plan 9 window manager 8½, also known as rio.
-It provides a very simple and clean user interface.
-It is click-to-type.
-It uses the X11 font system (which, unfortunately, means no Unicode support).
+9wm is an X11 window manager. It's inspired by the Plan 9 window manager 8½, also known as [rio].
 
-9wm does not provide virtual desktops, customization, key bindings,
-EWMH support, or compositing.
-It does not allocate any colors (if you disable COLOR),
-which will be great news if you are stuck in 1993.
+- Simple and clean user interface. It is click-to-type.
+- It uses the X11 font system (which, unfortunately, means no Unicode support).
+- No virtual desktops, customization, key bindings, EWMH support, or compositing.
+- It does not allocate any colors (if you disable COLOR), which will be great news if you are stuck in 1993.
 
-It is a great place to start if you are interested in writing a window manager from scratch:
-many folks have done exactly this.
+It is a great place to start if you are interested in writing a window manager from scratch: many folks have done exactly this.
 
-9wm is distributed under [an MIT License](LICENSE.md).
+`9wm` is distributed under [an MIT License](LICENSE.md).
 
+## Install
 
-Where do I get it?
-==================
+The latest version of 9wm is held at: <https://github.com/9wm/9wm>
 
-The latest version of 9wm is held at:
+You'll need `make`, an ANSI c compiler, `mkdir`, `cp`, `libX11` and `libXext` to build and install `9wm`. 
+In order to do so, run the following command:
 
-<https://github.com/9wm/9wm>
+```
+root # make install
+```
+You can launch `9wm` through a display manager like [lightdm] or via `startx` directly.
 
+To use `startx`, you'll need to include the following line at the end of your `.xinitrc`:
 
-How do I use it?
-================
+```sh
+exec 9wm # ... options
+```
 
-The focused window has a black border.
-Mouse and keyboard events go to this window.
-Left click a non-focused window (white border) to focus it.
+## Usage
 
-Right-click anywhere other than the focused window,
-including in any non-focused window,
-to bring up a menu.
-All further actions are done with the right mouse button.
-The menu has options for:
```

Comment:
I see what you did here in the rewrite, but the original text has the advantage of explaining immediately how to focus windows, and then explaining that everything else happens with the other mouse button. I think I prefer the original wording to the proposed change. I would like to hear from @Tookmund on this one.